src/system/catalog-sync.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Catalog Sync
+---
+
+Magento exports catalog data into tables using indexers. This process is automatically [triggered by several events]({% link system/index-management-events.md %}), such as changes to a product's price or inventory levels. Magento then syncs that catalog data to a central storage hub for use by various SaaS services, such as [Product Recommendations]({% link marketing/product-recommendations.md %}). The **Catalog Sync** dashboard allows you to peek into the Magento to SaaS syncing process and view the current status.
```

Comment:
Maybe give some more background as to why this is needed. 

Some Magento services require a Catalog Sync to be set up. This process exports product data from the Magento server to SaaS services on an ongoing basis. For example, the Product Recommendations feature need up-to-date catalog information so that it can accurately return recommendations with correct names, pricing, and availability.